### ISSUE TITLE
Skill ai

### ISSUE THREAD

**Issue by valanths1990** [L2J/L2J_Server#254]
Lets say we have an archer

if you auto attack and then press a skill example stun shot

interrupts hit and use skill

it should finish hit shot and then use skill

and gauge bar should disappear when use skill

Tested in off

**Comment by Zealar** [comment]
In l2j you cannot use skill while red line is on.

**Comment by valanths1990** [comment]
true, but player should be able to use skill while bar is at 50% or something

same how it allows him to move at 50%

ps. if you chnage this behavior, red gauge bar should not disappear(cancel) when use skill

**Comment by Zealar** [comment]
Yes in retail can start skill on like ~ 30% red left.

**Comment by valanths1990** [comment]
this can be fixed if you multiply bow attack end time
in L2Character onIntentionCast  * 0.7 for example

but i dont know if this will be proper fix
